dockerfiles/Dockerfile

```diff
@@ -12,6 +12,7 @@ ARG METASERVER_ID="1505"
 ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
 ARG MYSQL_CONNECTOR_VERSION="8.0.21"
 ARG HUDI_BUNDLE_VERSION="0.12.3.0"
+ARG TEZ_VERSION="0.9.1.5"
```

Comment:
you need to rebase since we just merged this PR

Reply:
Fixed